packages/explorable-explanations/src/protectedAudience/components/box.ts

```diff
@@ -99,7 +99,7 @@ const Box = ({
     p.text(title, x + width / 2, y + height / 2);
   }
 
-  if (info && typeof info === 'string') {
+  if (info) {
```

Comment:
Ah, we missed this in https://github.com/GoogleChromeLabs/ps-analysis-tool/pull/983 review.